src/witan/models/dem/ccm/models.clj

```diff
@@ -167,14 +177,18 @@
      :witan/fn :ccm-mig/proj-dom-out-mig
      :witan/params {:start-year-avg-domout-mig 2003, :end-year-avg-domout-mig 2014}}
     {:witan/name :project-asfr
-     :witan/version "1.0.0"
+     :witan/version "1.1.0"
+     :witan/type :function
+     :witan/fn :ccm-fert/project-asfr
+     :witan/params {:fert-variant :fixed, :first-proj-year 2015, :last-proj-year 2018,
+                    :fert-scenario :principal-2012}}
+    {:witan/name :project-asmr
+     :witan/version "1.1.0"
```

Comment:
I don't think so. Just like for asmr we set a default. Version 1.0.0 pointed to version 1.1.0. Version 1.0.0 has the open parameter set as a default, while 1.1.0 allows each variant